### ISSUE TITLE
Add Style Guide

### ISSUE THREAD

**Issue by brentleyjones** [TabletopAssistant/DiceKit#13]
Since I know I'm a stickler about code style we should have a style guide that we can point to when asking people to clean up contributions. A PR can still be merged without following it (if needed, like when a contributor goes silent), but only if someone on @TabletopAssistant/dicekit cleans it up right after.

**Comment by brentleyjones** [comment]
@TabletopAssistant/dicekit do we feel this is needed for release [0.1](https://github.com/TabletopAssistant/DiceKit/milestones/0.1:%20It%20Starts%20with%20a%20Single%20Die)?

**Comment by JonathanHoffman** [comment]
No, since 0.1 was originally defined as everything needed to be used by others. Contributing is not necessary for usage. :)

**Comment by brentleyjones** [comment]
Good call. This can get in whenever then, which is good, because I would rather get 0.1 and 0.2 out ASAP so I can start talking about the project more broadly... though I might get the style guide done before that because before people contribute I would want this in place (I think).

**Comment by LoganJohnson** [comment]
Also agreed here. Not needed yet but will be valuable eventually.
